big-practice/src/js/services/api.service.js

```diff
@@ -0,0 +1,92 @@
+class ApiService {
+  /**
+   * Get by calling API
+   */
+  async get(url) {
+    try {
+      // GET method implementation
+      const options = {
+        method: 'GET',
+      };
+
+      const res = await fetch(url, options);
+      const result = await res.json();
```

Comment:
Return res.json directly instead of using new variable. Should use variable in case you will it for handling anything after definition

Reply:
updated!